{
  "id": "uno-ooo-parser",
  "version": "0.2.0",
  "libre_office_ver": "7.3",
  "name": "InvocationTargetException",
  "type": "exception",
  "namespace": "com.sun.star.reflection",
  "parser_args": {
    "sort": false,
    "long_names": true,
    "remove_parent_inherited": true
  },
  "writer_args": {
    "include_desc": true
  },
  "data": {
    "allow_db": true,
    "from_imports": [
      [
        "..lang.wrapped_target_exception",
        "WrappedTargetException",
        "WrappedTargetException_38ae0f93"
      ]
    ],
    "from_imports_typing": [],
    "extends_map": {
      "com.sun.star.lang.WrappedTargetException": "WrappedTargetException_38ae0f93"
    },
    "quote": [],
    "typings": [],
    "requires_typing": false,
    "full_imports": {
      "general": [
        "com.sun.star.lang.WrappedTargetException"
      ],
      "typing": []
    },
    "name": "InvocationTargetException",
    "imports": [],
    "namespace": "com.sun.star.reflection",
    "extends": [
      "com.sun.star.lang.WrappedTargetException"
    ],
    "desc": [
      "This exception denotes a checked exception (wrapping an originating exception) and may be thrown upon using invocation API."
    ],
    "url": "https://api.libreoffice.org/docs/idl/ref/exceptioncom_1_1sun_1_1star_1_1reflection_1_1InvocationTargetException.html",
    "items": {}
  }
}